data/emea-on-rails/playlists.yml

```
@@ -0,0 +1,16 @@
+---
+- id: PL-pWnolRukJX1d7-cxY2jruggH47EplAe
+  title: EMEA on Rails 2021
+  location: Earth
```

Maybe we can update this to:
```suggestion
  location: online
```